src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -50,6 +50,9 @@
     <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.EnumProcessModules.cs">
       <Link>Common\Interop\Windows\kernel32\Interop.EnumProcessModules.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\NtDll\Interop.NtQuerySystemInformation.cs">
```

Comment:
The condition `'$(TargetGroup)' != 'uap'` is applied to exclude this file on Uap